test/gtest/uct/test_amo.h

```diff
@@ -79,6 +79,47 @@ class uct_amo_test : public uct_test {
         std::vector<completion> m_completions;
     };
 
+    ucs_status_t atomic_post(uct_ep_h ep, uct_atomic_op_t opcode,
+                             uint32_t value, uint64_t remote_addr,
+                             uct_rkey_t rkey) {
+        return uct_ep_atomic32_post(ep, opcode, value, remote_addr, rkey);
+    }
+
+    ucs_status_t atomic_post(uct_ep_h ep, uct_atomic_op_t opcode,
+                             uint64_t value, uint64_t remote_addr,
+                             uct_rkey_t rkey) {
+        return uct_ep_atomic64_post(ep, opcode, value, remote_addr, rkey);
+    }
+
+    ucs_status_t atomic_fetch_nb(uct_ep_h ep, uct_atomic_op_t opcode,
+                                 uint32_t value, uint32_t *result,
+                                 uint64_t remote_addr, uct_rkey_t rkey,
+                                 uct_completion_t *comp) {
+        return uct_ep_atomic32_fetch_nb(ep, opcode, value, result, remote_addr, rkey, comp);
+    }
+
+    ucs_status_t atomic_fetch_nb(uct_ep_h ep, uct_atomic_op_t opcode,
+                                 uint64_t value, uint64_t *result,
+                                 uint64_t remote_addr, uct_rkey_t rkey,
+                                 uct_completion_t *comp) {
+        return uct_ep_atomic64_fetch_nb(ep, opcode, value, result, remote_addr, rkey, comp);
+    }
+
+    template <typename T, uct_atomic_op_t opcode>
+    ucs_status_t op(uct_ep_h ep, worker& worker, const mapped_buffer& recvbuf,
```

Comment:
"op" -> "atomic_op"

Reply:
renamed (op/fop used to short lines :) )